govcd/adminorg.go

```diff
@@ -42,6 +42,74 @@ func (adminOrg *AdminOrg) CreateCatalog(name, description string) (AdminCatalog,
 	return CreateCatalog(adminOrg.client, adminOrg.AdminOrg.Link, name, description)
 }
 
+// CreateCatalogWithStorageProfile is like CreateCatalog, but allows to specify storage profile
+func (adminOrg *AdminOrg) CreateCatalogWithStorageProfile(name, description string, storageProfiles *types.CatalogStorageProfiles) (AdminCatalog, error) {
+	return CreateCatalogWithStorageProfile(adminOrg.client, adminOrg.AdminOrg.Link, name, description, storageProfiles)
```

Comment:
does this mean that catalog can have assigned multi storage profiles?

Reply:
Nope, but API  structure is such. The other way is I could always wrap it in this function. I just accepted this structure as it looked more convenient (and maybe future proof although I know it may never happen).

Reply:
Well due this naming and type I got strange impression that multi storage profiles supported:)

Reply:
Yes, the structure accepts a slice, but I left it as I see it as a possibility for future. If it can use "any of all", why can't it use "any of specified".